includes/Core/Assets/Assets.php

```diff
@@ -433,23 +433,23 @@ private function get_assets() {
 			new Script(
 				'googlesitekit-activation',
 				array(
-					'src'          => $base_url . 'js/googlesitekit-activation.js',
+					'version'      => null,
```

Comment:
Instead of setting `version` to `null` here throughout the file, let's handle this in the `Script` class automatically: If a file is in the manifest, no need to specify the version. If it is not in the manifest, we would specify the version.

Reply:
Updated.